### ISSUE TITLE
Add ContentLine import into docs

### ISSUE THREAD

**Issue by K3vX21** [ics-py/ics-py#377]
Maybe include `from ics.grammar.parse import ContentLine` in the [Advanced Usage](https://icspy.readthedocs.io/en/stable/advanced.html) section? Had to go to [ics.alarm.studio](https://icspy.readthedocs.io/en/stable/_modules/ics/alarm/audio.html) to find the import

**Comment by N-Coder** [comment]
Yep, docs will still need quite some work for the new version, thanks for bringing this up!
Could you also post your code for creating the repeating event in the other issue?
This might also be a nice example to have in the docs until we get first-class support.
